koku/api/provider/models.py

```diff
@@ -71,6 +71,7 @@ class Provider(models.Model):
                                  on_delete=models.CASCADE)
     created_by = models.ForeignKey('User', null=True,
                                    on_delete=models.SET_NULL)
+    report_processed = models.BooleanField(default=False)
```

Comment:
I agree it's not a great name.  Maybe `initial_setup_complete` or `setup_complete` to indicate that the account is healthy since it has downloaded and processed a report?

This is strictly internal.  With the default value provided.  I didn't think there would be any serializer change but I will double check.

Reply:
I'm fine with either of those names, maybe prefer `setup_complete`.